google-cloud-clients/google-cloud-bigquerydatatransfer/pom.xml

```diff
@@ -98,4 +71,4 @@
       </plugin>
     </plugins>
   </reporting>
-</project>
+</project>
```

Comment:
Here as well. Please check each PR and confirm about this line, I'll ignore this now

Reply:
already empty line is there but doesn't reflected it whether empty line is there on not so no need to add empty line in each PR.